packages/react-server-cli/src/compileClient.js

```diff
@@ -159,9 +160,13 @@ function statsToManifest(stats) {
 	};
 }
 
-function packageCodeForBrowser(entrypoints, outputDir, outputUrl, hot, minify, longTermCaching, stats) {
+function packageCodeForBrowser(entrypoints, outputDir, outputUrl, hot, minify, longTermCaching, stats, configPath) {
 	const NonCachingExtractTextLoader = path.join(__dirname, "./NonCachingExtractTextLoader");
 	const extractTextLoader = require.resolve(NonCachingExtractTextLoader) + "?{remove:true}!css-loader";
+	const babelrcPath = path.join(configPath || '', ".babelrc");
+	const babelConfig = configPath && fs.existsSync(babelrcPath) ?
```

Comment:
hmm... babel users can also put their [configuration in package.json](https://babeljs.io/docs/usage/babelrc/#use-via-package-json).  I _think_ we currently support that; does this change break that?